OneSignalSDK/onesignal/consumer-proguard-rules.pro

```diff
@@ -54,5 +54,3 @@
 -keep public class com.onesignal.ADMMessageHandler {*;}
 
 -keep class com.onesignal.JobIntentService$* {*;}
-
--keep class com.onesignal.OneSignalUnityProxy {*;}
```

Comment:
There is a specific Unity target where this file will exist so we need to keep this.

It might be possible to make a 2nd `consumer-proguard-rules.pro` for this target. Not sure if it will merge the main target with the unity target though.

A 3rd option is this proxy should be its own project completely or moved to the OneSignal-Unity-SDK repo.